python/lsst/obs/subaru/masking.py

```diff
@@ -0,0 +1,100 @@
+# This file is part of ip_isr.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+# import os
+
+import lsst.afw.geom as afwGeom
+import lsst.afw.image as afwImage
+import lsst.pex.config as pexConfig
+
+from lsst.ip.isr.masking import MaskingTask, MaskingConfig
+
+
+class SuprimeCamMaskingConfig(MaskingConfig):
+    suprimeMaskLimitSetA = pexConfig.Field(
+        dtype=int,
+        doc="CCD numbers for devices that have the 'near' limit.",
+        default=[1, 2, 7],
+    )
+    suprimeMaskLimitSetB = pexConfig.Field(
+        dtype=int,
+        doc="CCD numbers for devices that have the 'far' limit.",
+        default=[0, 6],
+    )
+
+
+class SuprimeCamMaskingTask(MaskingTask):
+    """Perform extra masking for detector issues such as ghosts and glints.
+    """
+    ConfigClass = SuprimeCamMaskingConfig
+
+    # CZW Finish conversion from guider to masking.run().  The main thing is setting ccdNumA/B entries.
```

Comment:
TODOs should have a ticket number.

Reply:
This was resolved in the ticket, but the comment not removed.  Cleared.